task1.java

```diff
@@ -0,0 +1,34 @@
+package com.Selenium.homework1;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class task1 {
+    public static void main(String[] args) {
+
+        List<String> l1 = new LinkedList<String>();

```

Comment:
1. variable name is not meaningful
2. you don't need to specify <String> in the second part, we have diamon syntax instead - '<>'

Reply:
fixed in 'task1.changes'